cajita/src/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ endif()
 
 if(Heffte_FOUND)
   target_link_libraries(Cajita Heffte::Heffte)
+  target_include_directories(Cajita PUBLIC ${Heffte_INCLUDE_DIR})
```

Comment:
Huh, isn't that part of `Heffte::Heffte`? (https://bitbucket.org/icl/heffte/src/458ecdc90e1d27e1cfd4d75bbee18b2a13f28a67/CMakeLists.txt#lines-91)

Reply:
Yes, I don't think adding this did anything helpful. Now I've removed it.